package.json

```diff
@@ -10,8 +10,8 @@
     "start": "yarn --cwd examples/sandbox start",
     "test": "lerna run test --stream --no-private",
     "test-ci": "lerna run test-ci --stream --no-private",
-    "watch": "lerna run watch --parallel",
-    "build": "lerna run build --stream --scope @elastic/search-ui && lerna run build --stream --scope '{@elastic/react-search-ui*,@elastic/search-ui-*}'",
+    "watch": "lerna exec --no-private -- yarn run watch",
```

Comment:
tried this with `run` and didn't work, realised you likely hit this too! I then learnt about differences between exec and run :joy: 

its surprising it includes the root `package.json` in the execution. If we do introduce any private packages, we can change it to an `--exclude root` flag.

Reply:
How do you know that it includes root `package.json` in the execution? My terminal gets cleared quickly and I can't check the logs :D

---

BTW, we do have private packages! examples/sandbox is considered private package because it's listed under packages: https://github.com/elastic/search-ui/blob/master/lerna.json#L2

Do you think we still need it there?